cumulus_library/studies/core/manifest.toml

```diff
@@ -23,7 +23,11 @@ file_names = [
     "observation_vital_signs.sql",
     "medication_request.sql",
     "study_period.sql",
-    "count_core.sql"
+]
+
+[count_table_builder_config]
```

Comment:
nit: you named the class `CountsBuilder` - maybe `counts_builder_config` would be a good symmetry? But no opinion myself

Reply:
good call